test/torchaudio_unittest/complex_batch_consistency_test.py

```diff
@@ -0,0 +1,49 @@
+"""Test numerical consistency among single input and batched input."""
+import unittest
+import itertools
+from parameterized import parameterized
+
+import torch
+import torchaudio
+import torchaudio.functional as F
+
+from torchaudio_unittest import common_utils
+
+
+class TestTransforms(common_utils.TorchaudioTestCase):
+    backend = 'default'
+
+    def test_batch_TimeStretch(self):
+        test_filepath = common_utils.get_asset_path('steam-train-whistle-daniel_simon.wav')
+        waveform, _ = common_utils.load_wav(test_filepath)  # (2, 278756), 44100
```

Comment:
BTW does the content of the audio matter here?
We are trying to avoid external audio files in new tests.
We have some utilities for generating synthetic data outlined [here](https://github.com/pytorch/audio/tree/master/test/torchaudio_unittest#waveforms-for-testing-purposes).
Sorry I should have thought of this in the first place.

Reply:
Also, if you decided to use synthetic data generated on-the-fly, please remove `backend='default'` above, so that torch audio's io function is used accidentally.

Reply:
I am not sure if it does. I just used the same logic as being used in the existing `test_batch_TimeStretch`.
btw I do see `steam-train-whistle-daniel_simon.wav` listed [here](https://github.com/pytorch/audio/tree/master/test/torchaudio_unittest#waveforms-for-testing-purposes) which is the what's being used in this test.

Reply:
I see. Could you then use synthetic data?

```
waveform = common_utils.get_whitenoise(n_channels=2)
```

should do. I will clean up and do the same for the original tests you referenced.